Sources/Atoms/Core/StoreContext.swift

```diff
@@ -454,6 +461,28 @@ private extension StoreContext {
         release(for: key)
     }
 
+    func detachDependencies(for key: AtomKey) -> Set<AtomKey> {
```

Comment:
aaah exactly what I wanted to ask (methods or closure to freeze updates to atom), thanks!